granary/reddit.py

```diff
@@ -1,215 +1,218 @@
 # coding=utf-8
 """reddit.com source class.
+
+reddit API docs:
+https://github.com/reddit-archive/reddit/wiki/API
+https://www.reddit.com/dev/apis
+praw API docs:
+https://praw.readthedocs.io/en/v3.6.0/pages/oauth.html
 """
 
 from . import source
-import datetime
 import logging
 from oauth_dropins import reddit
-from oauth_dropins.webutil import util
+from oauth_dropins.webutil import appengine_info, util
 import re
 import urllib.parse, urllib.request
 
 import praw
 
+if appengine_info.DEBUG:
+  REDDIT_APP_KEY = util.read('reddit_app_key_local')
+  REDDIT_APP_SECRET = util.read('reddit_app_secret_local')
+else:
+  REDDIT_APP_KEY = util.read('reddit_app_key')
+  REDDIT_APP_SECRET = util.read('reddit_app_secret')
+
+
+def get_reddit_api(refresh_token):
+  return praw.Reddit(client_id=REDDIT_APP_KEY,
```

Comment:
minor, mind grabbing the app key and secret from o-d instead? example:

https://github.com/snarfed/granary/blob/6e97787c72a139709e011587a65722ec64212fe4/granary/facebook.py#L914-L915

then you can drop loading them above.